engine/include/cubos/engine/audio/listener.hpp

```diff
@@ -0,0 +1,22 @@
+/// @file
+/// @brief Component @ref cubos::engine::AudioListener.
+/// @ingroup audio-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Component which adds an AudioListener to the entitiy
+    /// @ingroup audio-plugin
+
+    struct CUBOS_ENGINE_API AudioListener
+    {
+        CUBOS_REFLECT;
+
+        bool active; // is the listener active (assigned to an entity)
```

Comment:
```suggestion
        bool active; ///< Is the listener active?
```
It's not whether it is assigned to an entity or not - it's whether it's currently 'listening'.
Imagine you have a single-player game where you have two different cameras, and you toggle between them.
Each has their own audio listener, but only one is active at the same time - you don't want to get audio from both of them, only from the currently selected camera.